### ISSUE TITLE
502 Bad Gateway

### ISSUE THREAD

**Issue by Niklan** [dockerizedrupal/docker-apache-2.4#22]
Hi.

I trying to create new dockererizedrupal container but every time i get this message. No matter how i install it. Previously created works fine, this affected only by new one. No matter which PHP version i select to use, so i think this is apache issue.

My docker-compose.yml file.

``` yaml
# Configuration generated with Drupal Compose version 1.4.4
apache:
  extends:
    file: host.yml
    service: apache
  image: dockerizedrupal/apache-2.4:1.2.4
  hostname: apache
  volumes_from:
    - apache-data
  links:
    - php
  environment:
    - VHOST_PROJECT_NAME=uchinatraver
    - VHOST_PRIMARY_SERVICE=True
    - VHOST_SERVICE_NAME=apache
    - VHOST_VERSION_FILE_URL=https://raw.githubusercontent.com/dockerizedrupal/docker-apache-2.4/master/VERSION.md
    - VHOST_REPOSITORY_URL=https://hub.docker.com/r/dockerizedrupal/apache-2.4/
apache-data:
  image: dockerizedrupal/apache-2.4:1.2.4
  hostname: apache-data
  entrypoint: ["/bin/echo", "Data-only container for Apache."]
  volumes:
    - .:/apache/data
mysql:
  image: dockerizedrupal/mysql:1.2.4
  hostname: mysql
  volumes_from:
    - mysql-data
  environment:
    - VHOST_PROJECT_NAME=uchinatraver
    - VHOST_SERVICE_NAME=mysql
    - VHOST_VERSION_FILE_URL=https://raw.githubusercontent.com/dockerizedrupal/docker-mysql/master/VERSION.md
    - VHOST_REPOSITORY_URL=https://hub.docker.com/r/dockerizedrupal/mysql/
mysql-data:
  image: dockerizedrupal/mysql:1.2.4
  hostname: mysql-data
  entrypoint: ["/bin/echo", "Data-only container for MySQL."]
  volumes:
    - /mysql
php:
  extends:
    file: host.yml
    service: php
  image: dockerizedrupal/php-7.0:1.1.5
  hostname: php
  volumes:
    - ~/.ssh:/home/container/.ssh
  volumes_from:
    - apache-data
  links:
    - mysql
    - mailcatcher:smtp
    - memcached
  environment:
    - VHOST_PROJECT_NAME=uchinatraver
    - VHOST_SERVICE_NAME=php
    - VHOST_VERSION_FILE_URL=https://raw.githubusercontent.com/dockerizedrupal/docker-php-7.0/master/VERSION.md
    - VHOST_REPOSITORY_URL=https://hub.docker.com/r/dockerizedrupal/php-7.0/
mailcatcher:
  image: dockerizedrupal/mailcatcher:1.1.1
  hostname: mailcatcher
  environment:
    - VHOST_PROJECT_NAME=uchinatraver
    - VHOST_SERVICE_NAME=mailcatcher
    - VHOST_VERSION_FILE_URL=https://raw.githubusercontent.com/dockerizedrupal/docker-mailcatcher/master/VERSION.md
    - VHOST_REPOSITORY_URL=https://hub.docker.com/r/dockerizedrupal/mailcatcher/
phpmyadmin:
  image: dockerizedrupal/phpmyadmin:1.1.1
  hostname: phpmyadmin
  links:
    - mysql
  environment:
    - VHOST_PROJECT_NAME=uchinatraver
    - VHOST_SERVICE_NAME=phpmyadmin
    - VHOST_VERSION_FILE_URL=https://raw.githubusercontent.com/dockerizedrupal/docker-phpmyadmin/master/VERSION.md
    - VHOST_REPOSITORY_URL=https://hub.docker.com/r/dockerizedrupal/phpmyadmin/
adminer:
  image: dockerizedrupal/adminer:1.1.1
  hostname: adminer
  links:
    - mysql
  environment:
    - VHOST_PROJECT_NAME=uchinatraver
    - VHOST_SERVICE_NAME=adminer
    - VHOST_VERSION_FILE_URL=https://raw.githubusercontent.com/dockerizedrupal/docker-adminer/master/VERSION.md
    - VHOST_REPOSITORY_URL=https://hub.docker.com/r/dockerizedrupal/adminer/
memcached:
  image: dockerizedrupal/memcached:1.1.0
  hostname: memcached
  environment:
    - VHOST_PROJECT_NAME=uchinatraver
    - VHOST_SERVICE_NAME=memcached
    - VHOST_VERSION_FILE_URL=https://raw.githubusercontent.com/dockerizedrupal/docker-memcached/master/VERSION.md
    - VHOST_REPOSITORY_URL=https://hub.docker.com/r/dockerizedrupal/memcached/
memcachephp:
  image: dockerizedrupal/memcachephp:1.1.1
  hostname: memcachephp
  links:
    - memcached
  environment:
    - VHOST_PROJECT_NAME=uchinatraver
    - VHOST_SERVICE_NAME=memcachedphp
    - VHOST_VERSION_FILE_URL=https://raw.githubusercontent.com/dockerizedrupal/docker-memcachephp/master/VERSION.md
    - VHOST_REPOSITORY_URL=https://hub.docker.com/r/dockerizedrupal/memcachephp/
```

**Comment by Niklan** [comment]
Changed versions to working one and this helps me.

``` yaml
apache:
  image: dockerizedrupal/apache-2.4:1.2.4
apache-data:
  image: dockerizedrupal/apache-2.4:1.2.4
```

**Comment by viljaste** [comment]
Hi @Niklan,

Thanks for reporting this. New version 1.2.7 should resolve it.

**Comment by Niklan** [comment]
Problem solved in 1.2.7, so i close the issue.
